python/tvm/relay/expr.py

```diff
@@ -277,6 +277,12 @@ def set_params(self, params):
 
         return _expr.FunctionSetParams(self, params)
 
+    def is_primitive(self):
+        return int(self.get_attribute("Primitive")) == 1
```

Comment:
Move this as a auxiliary function instead of member for now. Because member function need to be relatively stable, and we might have new attributes later.

Reply:
Removed this function for now as it's not used.